addons/mail/wizard/mail_compose_message.py

```diff
@@ -413,6 +414,18 @@ def _compute_mail_server_id(self):
             if not composer.template_id:
                 composer.mail_server_id = False
 
+    @api.depends('composition_mode', 'model', 'res_ids', 'template_id')
+    def _compute_scheduled_date(self):
+        """ Computation is coming either from template, either reset. When
+        having a template with a value set, copy it (in batch mode) or render
+        it (in monorecord comment mode) on the composer. When removing the
+        template, reset it. """
+        for composer in self:
+            if composer.template_id:
+                composer._set_value_from_template('scheduled_date', force_void=False)
+            if not composer.template_id:
```

Comment:
Again, could be a else, or maybe remove this? Make sure to re-check all of them.